data/ui/application.ui

```diff
@@ -71,8 +71,8 @@
   <template class="Window" parent="GtkApplicationWindow">
     <property name="can_focus">False</property>
     <property name="window_position">center</property>
-    <property name="default_width">1400</property>
-    <property name="default_height">800</property>
+    <property name="default_width">1024</property>
```

Comment:
This is where the default window size is changed. I'd prefer this to not be the case.